Cargo.toml

```diff
@@ -40,6 +49,8 @@ image = "0.24.0"
 memoffset = "0.9.0"
 mint = "0.5.9"
 tobj = "4.0.0"
+ash = { version="0.37.1", default-features = false, features = ["linked", "debug"] }
```

Comment:
The linked feature of ash is only used for an example, so it got moved over here.